src/components/bottom-bar-item/styles.ts

```diff
@@ -0,0 +1,82 @@
+import { styled } from '@linaria/react'
+import { AnchorHTMLAttributes, ButtonHTMLAttributes } from 'react'
+
+type CommonBottomBarItemProps = {
+  /**
+   * NOTE: data attribute to indicate the state of the nav item
+   * by default the styles is already applied with pseudo-class selector
+   * but to make it more flexible, consumer could use this attribute to
+   * change the styles
+   */
+  'data-state'?: 'active'
+}
+
+export const ElBottomBarItemIcon = styled.div`
+  * {
+    width: var(--icon-default, 24px);
+    height: var(--icon-default, 24px);
+    color: var(--icon-secondary, #607890);
+  }
+`
+
+export const ElBottomBarItemLabel = styled.span`
+  color: var(--text-secondary, #607890);
+  text-align: center;
+
+  font-family: var(--font-family, Inter);
+  font-size: var(--font-size-2xs, 10px);
+  font-style: normal;
+  font-weight: 500;
+  line-height: var(--line-height-2xs, 12px); /* 120% */
+  letter-spacing: var(--letter-spacing-2xs, 0px);
+`
+
+const baseStyles = `
+  background-color: var(--white, #ffffff);
+  outline: none;
+  border: none;
+  display: flex;
+  cursor: pointer;
+  padding: var(--spacing-half, 2px) var(--spacing-none, 0px);
+  flex-direction: column;
+  align-items: center;
+  gap: var(--spacing-half, 2px);
+  
+
+`
+
+interface ElAnchorBottomBarItemProps extends AnchorHTMLAttributes<HTMLAnchorElement>, CommonBottomBarItemProps {}
+export const ElAnchorBottomBarItemContainer = styled.a<ElAnchorBottomBarItemProps>`
+  ${baseStyles}
+
+  &:active,
+  &[data-state='active'] {
+    ${ElBottomBarItemIcon} {
+      * {
+        color: var(--icon-action, #4e56ea);
+      }
```

Comment:
**question:** Is there a reason why you use `*` here, but in #191 you used `> *`?

Reply:
> Is there a reason why you use * here, but in https://github.com/reapit/elements/pull/191 you used > *?

it's because the `ElIcon` have their own color management; and at that time I am not aware if the semantic color variable between text and icon is different; but with the latest commit it should not use that pattern